ui/src/router/edit/EditRouterView.js

```diff
@@ -1,37 +1,100 @@
-import React, { useEffect } from "react";
+import React, { useContext, useEffect } from "react";
 import { TuringRouter } from "../../services/router/TuringRouter";
-import { FormContextProvider } from "../../components/form/context";
-import { replaceBreadcrumbs } from "@gojek/mlp-ui";
+import {
+  FormContext,
+  FormContextProvider,
+} from "../../components/form/context";
+import { addToast, replaceBreadcrumbs, useToggle } from "@gojek/mlp-ui";
 import { UpdateRouterForm } from "../components/form/UpdateRouterForm";
 import { ExperimentEngineContextProvider } from "../../providers/experiments/ExperimentEngineContextProvider";
+import { VersionComparisonView } from "./components/VersionComparisonView";
+import { useTuringApi } from "../../hooks/useTuringApi";
+import { DeploymentSummary } from "../components/form/components/DeploymentSummary";
+import { ConfirmationModal } from "../../components/confirmation_modal/ConfirmationModal";
+
+const EditRouterView = ({ projectId, currentRouter, ...props }) => {
+  const { data: updatedRouter } = useContext(FormContext);
+  const [showDiffView, toggleDiffView] = useToggle();
```

Comment:
Can the `StepsWizardHorizontal` not be reused for this (hiding the `EuiStepsHorizontal` if needed)?

Reply:
That's what I initially started exploring, but then dropped, because it will break the layout and there is no a simple way to customize it:

_Current:_
<img width="1280" alt="Screenshot 2021-09-07 at 08 43 39" src="https://user-images.githubusercontent.com/1886194/132291182-b80f07cb-5be7-4d6c-a063-5d337e626f0c.png">

_With `StepsWizardHorizontal`:_
<img width="1279" alt="Screenshot 2021-09-07 at 08 45 32" src="https://user-images.githubusercontent.com/1886194/132291194-ab5be843-149f-4d94-bcc3-99fad0d790a0.png">

Reply:
I see.. Ok, let's keep it the way it is now.